### PR TITLE
feat: add timezone support

### PR DESCRIPTION
**Pull request by xjasonlyu**

### CODE REVIEW

**Review by ngjaying — CHANGES_REQUESTED** [review]
What is the default timezone value? Local? Need to add that to the default kuiper.yaml explicitly.  Also, need to add the doc.

**Comment by xjasonlyu** [comment]
> What is the default timezone value? Local? Need to add that to the default kuiper.yaml explicitly. Also, need to add the doc.

The default timezone is UTC. This will prevent behavior changes due to different system’s timezone settings. 

I will open another PR for docs.

File: docs/zh_CN/configuration/global_configurations.md [hunk 1]
```diff
@@ -44,6 +44,15 @@ basic:
...
 用户将名为 KuiperSyslogKey 的环境变量的值设置为 true 时，日志将打印到系统日志中。
 
+## 时区配置
+
+```yaml
+# The global time zone from the IANA time zone database, or UTC if not set.
+timezone: UTC
+```
+
+基于 IANA 时区数据库的全局时区配置，如果留空则使用 UTC 作为默认时区，设置为 Local 时则使用系统时区。
```
Comment (ngjaying) [inline docs/zh_CN/configuration/global_configurations.md:54]: Are there links for IANA?  Is `Local` already supported in the code, I don't find any special treatment for the value.

Comment (xjasonlyu) [reply docs/zh_CN/configuration/global_configurations.md:54]: IANA link: https://www.iana.org/time-zones

`Local` is supported by Golang std time pkg:

> Local represents the system's local time zone. On Unix systems, Local consults the TZ environment variable to find the time zone to use.

**Review by ngjaying — APPROVED** [review]

**Pull request merged**